### PR TITLE
docs: add changelog and complete documentation

### PR DESCRIPTION
**Pull request by JuanDavidBuitrago**
Description:
This PR adds repository documentation

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
+Codejailservice
```
Comment (mariajgrimaldi) [inline README.rst]: Codejail service?

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+|ci-badge| |license-badge|
+
+The repository codejailservice manages execution of
+untrusted code in secure sandboxes. It is designed primarily for Python execution, but
+can be used for other languages as well. The security is enforced with AppArmor.
+
+Usage
+-----
+
+To use with tutor plugin
```
Comment (mariajgrimaldi) [inline README.rst]: Using with Tutor (link to tutor) plugin: tutor-contrib-codejail (link)

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the instructions from `tutor-contrib-codejail`_
+
+.. _tutor-contrib-codejail: https://github.com/eduNEXT/tutor-contrib-codejail
+
+To use service
```
Comment (mariajgrimaldi) [inline README.rst]: Using service independently

File: docs/decisions/0001-purpose-of-this-repo.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+-------
+
+Codejailservice is a service that manages execution of untrusted code in secure sandboxes. It's useful
+because allows executing code outside of ``edx-platform``, avoiding security risks on servers, limiting
+staff permissions in some Open edx releases.
+
+Decision
+--------
+
+Create a codejailservice that manages the execution of untrusted code in secure sandboxes.
```
Comment (mariajgrimaldi) [inline docs/decisions/0001-purpose-of-this-repo.rst]: create a codejail independent service...

File: docs/decisions/0001-purpose-of-this-repo.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+------
+
+Accepted
+
+Context
+-------
+
+Codejailservice is a service that manages execution of untrusted code in secure sandboxes. It's useful
+because allows executing code outside of ``edx-platform``, avoiding security risks on servers, limiting
+staff permissions in some Open edx releases.
```
Comment (mariajgrimaldi) [inline docs/decisions/0001-purpose-of-this-repo.rst:14]: I like this for the repo description instead of what is now

File: CHANGELOG.rst [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+* Second git tagged release
+
+[11.0.0] - 2021-06-02
+~~~~~~~~~~~~~~~~~~~~~
+
+Added
+_____
+
+* First git tagged release
```
Comment (mariajgrimaldi) [inline CHANGELOG.rst]: Change this to: First git tagged release with service functionality

File: CHANGELOG.rst [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+* Add requirements files.
+* Add Add required maintenance files.
+
+[12.0.1] - 2021-08-19
+~~~~~~~~~~~~~~~~~~~~~
+
+Added
+_____
+
+* Third git tagged release
```
Comment (mariajgrimaldi) [inline CHANGELOG.rst]: Add what was changed from 12.0.1 to here

File: CHANGELOG.rst [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+[12.0.0] - 2021-08-19
+~~~~~~~~~~~~~~~~~~~~~
+
+Added
+_____
+
+* Second git tagged release
```
Comment (mariajgrimaldi) [inline CHANGELOG.rst]: What changed? Lilac support maybe. Check this

Comment (JuanDavidBuitrago) [reply CHANGELOG.rst]: I not found changed between 12.0.0 and 12.0.1 

https://github.com/eduNEXT/codejailservice/compare/12.0.1...12.0.0

File: README.rst [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+The creation of Codejail service affects the way untrusted code is executed, because it runs in a separate
+container from the one running the platform.
+
+Usage
+-----
+
+Using with `Tutor <https://docs.tutor.overhang.io>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Plugin: tutor-contrib-codejail (https://github.com/eduNEXT/tutor-contrib-codejail)
```
Comment (mariajgrimaldi) [inline README.rst:20]: Can you add it as a link? using `  ` __

Also, point that they should follow the instructions indicated in the plugin

**Review by mariajgrimaldi — APPROVED** [review]

File: README.rst (hunk 9 above)
Comment (JuanDavidBuitrago) [reply README.rst:20]: Done

**Pull request merged**